app/static/js/common.v2.2.js

```diff
@@ -52,6 +52,12 @@ function randomChoice(array) {
   }
 }
 
+//Choose a random element from an array
```

Comment:
Doing this is a little bit controversial. If a future version of javascript creates a `selectRandom` array method, then this will overwrite it. I think this is unlikely and it is much more convenient than having a separate method.

Reply:
sounds good to me